src/types.js

```diff
@@ -21,10 +21,11 @@ export type Message = {
   severity: 'error' | 'warning' | 'info',
   reference: ?string,
   solutions?: Array<{
-    title: ?string,
-    position: Range,
-    currentText: ?string,
-    replaceWith: string,
+    title?: string,
+    position?: Range,
+    currentText?: string,
+    replaceWith?: string,
```

Comment:
See https://github.com/steelbrain/linter/pull/1206/files#r71568991 for the problem with this.